android/bugstr-nostr-crypto/src/main/kotlin/com/bugstr/nostr/crypto/Nip17CrashSender.kt

```
@@ -1,48 +1,289 @@
+        return if (payloadSize <= Transport.DIRECT_SIZE_THRESHOLD) {
+            sendDirect(request)
+        } else {
+            sendChunked(request, onProgress)
+        }
+    }
+
+    private suspend fun sendDirect(request: Nip17SendRequest): Result<Unit> {
+        // Wrap in DirectPayload format
+        val crashJson = try {
+            JSONObject(request.plaintext)
+        } catch (e: org.json.JSONException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)